Dockerfile

```diff
@@ -46,12 +46,6 @@ RUN for pem_file in .ssh/jenkins-*; do \
       ssh-keygen -f "$pem_file" -y > "/root/.ssh/$(basename "$pem_file").pub"; \
     done
 
-RUN if [[ -z '$EXTERNAL_ENCODED_VPN' ]] ; then \
-      echo 'no vpn provided' ; \
```

Comment:
Why is this being removed?

Reply:
no longer needed to work with the new VPN system in jenkins